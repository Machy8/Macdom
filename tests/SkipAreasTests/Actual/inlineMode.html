<style>h1 { color: #123;}</style>

div <?php $a = 123; echo $a; ?>

a <?php $a = 123; echo $a; ?>

<script type="text/javascript"> var a = 123; a ++; alert(a);</script>

<script> var a = 123; a ++; alert(a); </script>

<macdom-off> var a = 123; a ++; alert(a);</macdom-off>
