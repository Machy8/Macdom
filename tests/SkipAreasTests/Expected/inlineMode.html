<style>
	h1 { color: #123;}
</style>
<div>
	<?php $a = 123; echo $a; ?>
</div>
<a>
	<?php $a = 123; echo $a; ?>
</a>
<script type="text/javascript">
	 var a = 123; a ++; alert(a);
</script>
<script>
	 var a = 123; a ++; alert(a); 
</script>
var a = 123; a ++; alert(a);
